gamemodes/terrortown/entities/entities/ttt_flame.lua

```diff
@@ -13,54 +13,74 @@ ENT.Model = Model("models/weapons/w_eq_flashbang_thrown.mdl")
 -- @realm shared
 AccessorFunc(ENT, "dmgparent", "DamageParent")
 
----
--- @accessor boolean
--- @realm shared
-AccessorFunc(ENT, "die_explode", "ExplodeOnDeath")
+ENT.firechild = nil
 
----
--- @accessor number
--- @realm shared
-AccessorFunc(ENT, "dietime", "DieTime")
+ENT.fire_ignite_delay = 0
```

Comment:
This was changed again because the flame already builds up in size as the particle system begins emitting -- but the hurtbox does not scale over time, so you will be hurt by something with no corresponding visual.

The engine code doesn't really make much use of this, but I pass it through in the off chance that someone overwrites the "env_fire" entity with a better implementation that uses the engine parameters as documented.

Reply:
Since this is just a new variable, lets just remove it. This would be a very niche use case and then someone could still overwrite the function and do so.

Reply:
The variable isn't new exactly, it's just externalized from its original position.

Reply:
Yes but its nothing that someone would already rely on and we do not really need to expose this. This is a really hypothetical usecase so i suggest to just remove it